### PR TITLE
feature - removed address behaviours

### PR DESCRIPTION
**Pull request by joefitter**
* installed hof-behaviour-address-lookup
* use address lookup for address pages
* removed superfluous translations and config

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,12 +29,13 @@
     "browserify": "browserify ./assets/js/index.js > ./public/js/bundle.js",
     "create:public": "mkdir -p ./public/js ./public/css ./public/images",
     "sass": "node-sass ./assets/scss/app.scss ./public/css/app.css --include-path ./node_modules",
-    "hof-transpile": "hof-transpiler ./apps/**/translations/src -w --shared ./node_modules/hof-template-partials/translations",
```
Comment (lennym) [inline package.json]: If you're removing this then I'd also bump the `hof-bootstrap` version to latest as it needs *at least* v10.3.0 to work.

Comment (joefitter) [inline package.json]: Ja good point

**Comment by joefitter** [comment]
@lennym any _service specific_ acceptance tests were left, I removed all the stuff that's tested in the behaviour, as I didn't see the benefit of duplicating stuff. If you think they should be left I'll leave them

**Comment by lennym** [comment]
@joefitter If you take that route then 99% of our acceptance tests are redundant because they're just re-testing that hof works as hof should. However, I think they still have value because they leave us flexible to shift implementations while maintaining confidence that our projects remain working as intended.

**Comment by joefitter** [comment]
@lennym fair, I'll reinstate them. When writing a new service though would you recommend adding similarly thorough acceptance tests? Say service (such as this) has 3 address lookups, we are testing the same code works as intended 3 times

**Comment by lennym** [comment]
"In theory" the acceptance tests should be independent of implementation, and so "yes". In practice "it depends".

**Pull request merged**